bin/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Tag, Push and Deploy only if it's not a pull request
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    # Push only if we're testing the staging branch
+    #if [ "$TRAVIS_BRANCH" == "staging" ]; then
+
+        export PATH=$PATH:$HOME/.local/bin
+
+        echo Getting the ECR login...
+        eval $(aws ecr get-login --region $AWS_DEFAULT_REGION)
```

Comment:
We'll need to add `--no-include-email` parameter to the `aws ecr get-login` command - I've experienced breaking errors otherwise

Reply:
Yes that was an oversight- `--no-include-email` is definitely necessary. Will add back in.